transforms/he/he_capability2activity.py

```diff
@@ -3,16 +3,16 @@
     'sheet': 'cap2activity',
     'lets': {
         'iri': 'vm:HE/{row[CapabilityID].as_slug}-{row[ActivityID].as_slug}',
-        'parent_iri': 'vm:HE/{row[CapabilityID].as_slug}',
         'activity_iri': 'vm:HE/{row[ActivityID].as_slug}',
     },
+    'allow_empty_subject': True,
     'non_unique': ['vm:hasInvolvement'],
     'triples': [
         ('{iri}', 'rdf:type', 'vm:HE/ActivityInvolvement'),
         ('{iri}', 'vm:description', '{row[Description].as_text}'),
         ('{iri}', 'vm:name',
             '{row[CapabilityID].as_slug}-{row[ActivityID].as_slug}'),
-        ('{parent_iri}', 'vm:hasInvolvement', '{iri}'),
+        ('vm:HE/{row[CapabilityID].as_slug}', 'vm:hasInvolvement', '{iri}'),
```

Comment:
Just going to push back gently on this and the other similar change -- yes, from a purely programming point of view aliasing a thing that is only used once is a waste of lines and characters. However, we are constructing a complex web of nodes here, and I found that using `lets` as a way of stating which classes were involved in this sheet and/or transform to be extremely helpful in keeping track of what was going on and what the relationships between classes were. 

Perhaps that's just me and when I've been working with triples and graphs a bit longer I won't need the hint, but for now I'd prefer we keep the aliasing.